Detectors/Upgrades/ALICE3/FT3/simulation/include/FT3Simulation/FT3Layer.h

```diff
@@ -0,0 +1,68 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file FT3Layer.h
+/// \brief Definition of the FT3Layer class
+
+#ifndef ALICEO2_FT3_UPGRADEV3LAYER_H_
+#define ALICEO2_FT3_UPGRADEV3LAYER_H_
+
+#include <TGeoManager.h>            // for gGeoManager
+#include "Rtypes.h"                 // for Double_t, Int_t, Bool_t, etc
+#include "FT3Simulation/Detector.h" // for Detector, Detector::Model
+
+class TGeoVolume;
+
+namespace o2
+{
+namespace ft3
+{
+
+/// This class defines the Geometry for the FT3 Layer TGeo. This is a work class used
+/// to study different configurations during the development of the ALICE3 EndCaps
+class FT3Layer : public TObject
+{
+ public:
+  // Default constructor
+  FT3Layer() = default;
+
+  // Sample layer constructor
+  FT3Layer(Int_t layerDirection, Int_t layerNumber, std::string layerName, Float_t z, Float_t rIn, Float_t rOut, Float_t sensorThickness, Float_t Layerx2X0);
+
+  /// Copy constructor
+  FT3Layer(const FT3Layer&) = default;
+
+  /// Assignment operator
+  FT3Layer& operator=(const FT3Layer&) = default;
+
+  /// Default destructor
+  ~FT3Layer() = default;
```

Comment:
```suggestion
  ~FT3Layer() override = default;
```
Should fix fullCI

Reply:
Yes. I am testing locally... starting to adopt the [Full CI test](https://aliceo2group.github.io/quickstart/devel.html) before pushing: (`ALIBUILD_O2_TESTS=1 aliBuild ...`).